change/@fluentui-react-text-bc1583ee-95cd-4270-a0e2-87e954a59d21.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "patch",
```

Comment:
why are there 2 change files for the same package ?

Reply:
Will remove it, looks strange 😨